profiles/permissions_test.py

```diff
@@ -204,20 +204,51 @@ def test_can_view_if_verified_mm_user(self):
         perm = CanSeeIfNotPrivate()
         with mute_signals(post_save):
             ProfileFactory.create(user=self.profile_user, account_privacy=Profile.PUBLIC_TO_MM)
+            program = ProgramFactory.create()
+            ProgramEnrollment.objects.create(
+                program=program,
+                user=self.profile_user,
+            )
 
         with mute_signals(post_save):
-            verified_user = UserFactory.create()
-            username = "{}_edx".format(verified_user.username)
-            verified_user.social_auth.create(
+            user = UserFactory.create()
+            username = "{}_edx".format(user.username)
+            user.social_auth.create(
```

Comment:
This is out of scope, i think we should handle this in a separate task.
please create an issue, i am happy to take that task